app/app.go

```diff
@@ -362,8 +363,10 @@ type App struct {
 	EIBCKeeper      eibckeeper.Keeper
 
 	// this line is used by starport scaffolding # stargate/app/keeperDeclaration
-	DelayedAckKeeper    delayedackkeeper.Keeper
-	DenomMetadataKeeper *denommetadatamodulekeeper.Keeper
+	DelayedAckKeeper     delayedackkeeper.Keeper
+	DenomMetadataKeeper  *denommetadatamodulekeeper.Keeper
+	TransferInjectKeeper *transferinjectkeeper.Keeper
```

Comment:
u don't need two transferKeepers
u probably need the IBC transfer keeper, and pass it your middleware as ICS4 wrapper